proto/src/main/proto/Rss.proto

```diff
@@ -151,6 +151,8 @@ message ShuffleRegisterRequest {
   int32 shuffleId = 2;
   repeated ShufflePartitionRange partitionRanges = 3;
   RemoteStorage remoteStorage = 4;
+  string user = 5;
```

Comment:
Could we put user and securityEnable into the field `remoteStorage`?

Reply:
Sounds great! I will do it.

Besides it looks we dont need to populate securityEnabled, it can be inferred by remote conf.